prow/github/client.go

```diff
@@ -386,6 +386,24 @@ func (c *Client) CreateStatus(org, repo, ref string, s Status) error {
 	return err
 }
 
+func (c *Client) GetLabels(org, repo string) ([]string, error) {
```

Comment:
I don't think it works the same way.  AddLabel simply creates a new label.  Assigning people obviously doesn't create new people if they don't exist nor does it assign if member is not part of the org.